cmd/account/reset.go

```diff
@@ -127,7 +139,24 @@ func (o *resetOptions) run() error {
 	if o.resetLegalEntity {
 		account.Spec.LegalEntity = v1alpha1.LegalEntity{}
 	}
-
+	var rootId string
+	if o.payerAccount == "osd-staging-1" {
+		rootId = OSDStaging1RootID
+	} else if o.payerAccount == "osd-staging-2" {
+		rootId = OSDStaging2RootID
+	} else {
+		return fmt.Errorf("invalid payer account provided")
```

Comment:
Given that we explicitly only support 2 payer account values, it would be more user friendly to include that in the error message. I might suggest something along the lines of:
`invalid payer account provided. Currently supported: osd-staging-1, osd-staging-2`

Reply:
yeah that's a great way to do it!

Reply:
The payer account should be known from the hive, so I actually don't think we need this check